scalableminds/webknossos
# Disallow deactivating oneself

( Note that this also includes pr #5073 Run snapshot tests without concurrency for deterministic ordering )

### Steps to test:
- As admin or team manager, go to user page, try to deactivate yourself, should get readable error
- Should still be allowed to deactivate other users

### Issues:
- fixes #5061 
- fixes #3798


------
